packages/block-library/src/post-excerpt/index.php

```diff
@@ -8,14 +8,39 @@
 /**
  * Renders the `core/post-excerpt` block on the server.
  *
+ * @param array $attributes The block attributes.
+ *
  * @return string Returns the filtered post excerpt for the current post wrapped inside "p" tags.
  */
-function render_block_core_post_excerpt() {
+function render_block_core_post_excerpt( $attributes ) {
 	$post = gutenberg_get_post_from_context();
 	if ( ! $post ) {
 		return '';
 	}
-	return '<p>' . get_the_excerpt( $post ) . '</p>';
+
+	$more_text = isset( $attributes['moreText'] ) ? '<a href="' . esc_url( get_the_permalink( $post ) ) . '">' . $attributes['moreText'] . '</a>' : '';
+
+	$filter_excerpt_length = function() use ( $attributes ) {
+		return isset( $attributes['wordCount'] ) ? $attributes['wordCount'] : 55;
+	};
+	add_filter(
+		'excerpt_length',
+		$filter_excerpt_length
+	);
+
+	$output = '<p>' . get_the_excerpt( $post );
```

Comment:
This element should have a class name, so that it can be targeted by Theme and Plugin developers.